src/evaluation/modisco/run.sh

```diff
@@ -12,27 +12,15 @@ output_dir=$2
 score_type=$3
 seqlets=$4
 crop=$5
-#meme_db=$6
-#meme_logos=$7
-#vier_logos=$8
-#vier_html=$9
-#html_link=${10}
-
-scores_prefix=/oak/stanford/groups/akundaje/projects/chrombpnet_paper/testing/testing
-output_dir=/oak/stanford/groups/akundaje/projects/chrombpnet_paper/testing/
-score_type=counts
-seqlets=50000
-crop=1000
-meme_db=/oak/stanford/groups/akundaje/soumyak/motifs/motifs.meme.txt
-meme_logos=/oak/stanford/groups/akundaje/soumyak/motifs/pfms/
-vier_logos=/oak/stanford/groups/akundaje/projects/chromatin-atlas/vierstra_logos/
-
-vier_html=http://mitra.stanford.edu/kundaje/anusri/chrombpnet_paper/vierstra_logos/
-html_link=http://mitra.stanford.edu/kundaje/anusri/chrombpnet_paper/testing/
+meme_db=$6
```

Comment:
Suggestion to add default values if the argument is not provided 
```suggestion
meme_db=${6:-/oak/stanford/groups/akundaje/soumyak/motifs/motifs.meme.txt}
meme_logos=${7:-/oak/stanford/groups/akundaje/soumyak/motifs/pfms/}
vier_logos=${8:-/oak/stanford/groups/akundaje/projects/chromatin-atlas/vierstra_logos/}
vier_html=${9:-http://mitra.stanford.edu/kundaje/anusri/chrombpnet_paper/vierstra_logos/}
html_link=${10:-http://mitra.stanford.edu/kundaje/anusri/chrombpnet_paper/testing/}
```